docs/serving/app-security/security-guard-install.md

```diff
@@ -61,15 +61,153 @@ To start this tutorial, after installing Knative Serving, run the following proc
         ```
         kubectl apply -f https://github.com/knative-sandbox/security-guard/releases/download/v0.3.0/guard-service.yaml
         ```
+=== "Install using the Knative Operator"
+
+    Use the following script to install Security-Guard and Serving using the Knative Operator:
+
+    ```
+    kubectl apply --filename - <<EOF
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: knative-serving
+    ---
+    apiVersion: operator.knative.dev/v1beta1
+    kind: KnativeServing
+    metadata:
+      name: knative-serving
+      namespace: knative-serving
+    spec:
+      security:
+        securityGuard:
+          enabled: true
+      ingress:
+        kourier:
+          enabled: true
```

Comment:
This is the configuration to use kourier as the ingress. We need to indicate that.
It is possible that users use istio or contour as the ingress, then the configuration is different.